plugins/restful/RestfulEntityBase.php

```diff
@@ -665,6 +735,15 @@ protected function setPropertyValues(EntityMetadataWrapper $wrapper, $null_missi
     $save = FALSE;
     $original_request = $request;
 
+    $default_language = language_default();
+    $title_field = $this->getTitleField();
+
+    // Set the default language of a new instance if this bundle support
+    // field translation.
+    if ($this->getMethod() == \RestfulInterface::POST && entity_translation_node_supported_type($this->getBundle())) {
```

Comment:
> Do we want to re-use the `is write request` helper method (forgot the exact name in 1.x) instead of `$this->getMethod() == \RestfulInterface::POST`?

I think that we should set the instance default language only on creation.